src/backend/index.js

```diff
@@ -43,6 +43,15 @@ async function updateFeed(feed) {
   return currentFeed;
 }
 
+/**
+ * Invalidates a feed
+ * @param {Feed} feed
```

Comment:
This is wrong.  `feedData` is a regular JS Object with properties of a feed, but not a Feed object.  Because we process feeds in a separate process, they get passed across process boundaries as JSON.